templates/aboutUsSection.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="container1">
+  <h2 class="eyebrow">
+    {{eyebrow}}
+  </h2>
+    <h3 class="title3">
+      {{title}}
+    </h3>
+      <p class="description">
+        {{description}}
+      </p>
+</div>
+
+<div class="container2">
+  <div class="cards-grid__container">
+  {{#each personCard}}
+    {{{module this}}}
+  {{/each}}
+  </div>
+</div>
+
+<div class="quotation-marks__container">
+  <div class="quotation-marks1">

```

Comment:
I would suggest a class name like quotation-marks__open and quotation-marks__close to improve naming!